docs/source/models/cause.rst

```diff
@@ -255,13 +254,13 @@ and discussed in more detail afterward.
      - Rate at which an individual accrues years lived with disability due to
        the state in the cause model.
    * - `Cause-specific Mortality`_
-     -
-     -
-     -
+     - Mortality rate from a specified cause for a population
+     - Mortality Impacts
+     - Measure of the frequency of occurrence of death in a defined population from a specified cause
    * - `Excess Mortality`_
-     -
-     -
-     -
+     - A temporary increase in the mortality rate (number of deaths) in a given population
+     - Mortality Impacts
+     - Estimate of the excess risk of death for an individual in the total population imposed by the disease studied
```

Comment:
The increase in the cause deleted mortality rate experienced by people living with the cause.